library/std/src/sys/unix/fs.rs

```diff
@@ -321,14 +321,19 @@ impl FileAttr {
         }))
```

Comment:
The line above here (just out of reach of GitHub's review system) seems to suggest that maybe instead of holding a timespec we could hold a seconds and nanoseconds field and then easily re-use the common implementation here.

Though if the libc test says we should use a timespec, that's fine too.

Reply:
I forget the exact result from the test, let me try again and see. Consistency with other targets would probably be nicer if possible

Reply:
Ok, I found that some other targets effectively do some "renaming" of the the fields: https://github.com/Meziu/libc/blob/master/libc-test/build.rs#L3786

Based on that, I'm not sure which is more desirable, given that is a `FIXME` I'm not sure if they want to fix the test to include the unpacked fields, or if they want to rename the fields (would be a breaking change, presumably?)

Also, since this PR merged already, maybe just let it ride and see how libc maintainers feel when we upstream it.

Reply:
It looks like only that one "Haiku" target does that renaming, so I'm fine with leaving it as is and seeing what the libc maintainers think.